### PR TITLE
Events schema

### PR DESCRIPTION
**Pull request by Suuuuha**
related to #94

### CODE REVIEW

File: database/events_schema.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+const mongoose = require('mongoose');
+
+const eventsSchema = mongoose.Schema({
+  eventTitle: {type: String, required: true},
+  eventDate: {type: Date, min: Date.now(), required},
```
Comment (Karyum) [inline database/events_schema.js]: what is min again?

File: database/events_schema.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  eventTime: {type: Number, required},
```
Comment (Karyum) [inline database/events_schema.js]: it should be required: true

File: database/events_schema.js (hunk 1 above)
Comment (Suuuuha) [inline database/events_schema.js]: minimum date they can type in, it should be minimum the date of the current day they add the event in

File: database/events_schema.js (hunk 2 above)
Comment (Suuuuha) [inline database/events_schema.js]: yea sorry missed that, changed

File: database/events_schema.js (hunk 1 above)
Comment (Karyum) [inline database/events_schema.js]: ohhh okay

**Comment by esraajb** [comment]
@Suuuuha @Karyum this one is failing

**Comment by esraajb** [comment]
@Karyum merge conflict

**Comment by Karyum** [comment]
@esraajb conflict and travis were fixed

**Pull request merged**